src/api/ops-transformations.js

```diff
@@ -135,6 +135,9 @@ function center (axis, ...objects) { // v, obj or array
       object = object.union(_objects[i])
     }
   }
+  if (axis.constructor !== Array) {
```

Comment:
from what I have seen , apparently the best way to test for an Array is `Array.isArray()`

Reply:
Yup. I did the research too.
We should be consistent going forward.